src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.PendingBranchesCollection.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal partial class AbstractFlowPass<TLocalState, TLocalFunctionState>
+    {
+        internal sealed class PendingBranchesCollection
+        {
+            private ArrayBuilder<PendingBranch> _unlabeledBranches;
+            private int _nextLabelIndex;
```

Comment:
Seems unnecessary. All uses seem like they could be replaced with `_labeledBranches.Count`